graphite/server.sls

```diff
@@ -11,6 +11,16 @@ graphite_packages:
     - graphite-web
     - python-txamqp
 
+{%- if grains.osfinger == "Ubuntu-16.04" %}
+
+graphite_django:
+  pip.installed:
```

Comment:
graphite-web is coming from Debian package so it should ship proper version of Django. What's the problem here?